TypeInjections/Tool/Tool.fs

```diff
@@ -11,6 +11,33 @@ open CommandLine
 
 module Tool =
 
+
+    [<Verb("local", HelpText = "[old project path] [new project path] [outputPath]")>]
+    type LocalOptions =
+        { [<Option('o', "old", Required = false, HelpText = "folder path of old project")>]
+          oldFolder: string
+          [<Option('n', "new", Required = false, HelpText = "folder path of new project")>]
+          newFolder: string
+          [<Option('p', "output", Required = false, HelpText = "folder path to place typeCart files")>]
+          outputFolder: string
+          [<Option('f', "file", Required = false, HelpText = "specify filename you want to run typecart on")>]
+          file: seq<string> }
+
+    [<Verb("git", HelpText = "[old commit] [new commit] [outputPath]")>]
+    type GitOptions =
+        { [<Option('o', "old", Required = true, HelpText = "commit hash of old version")>]
+          oldHash: string
+          [<Option('n', "new", Required = true, HelpText = "commit hash of new version")>]
+          newHash: string
+          [<Option('p', "output", Required = true, HelpText = "folder path to place typeCart files")>]
+          outputFolder: string
+          [<Option('f', "file", Required = false, HelpText = "specify filename you want to run typecart on")>]
```

Comment:
the shorthand `f` is duplicated, can we rename this to `entrypoint`?

Reply:
fixed